model-per-category/create-category-models/script.whizzml

```diff
@@ -40,7 +41,8 @@
 ;; Final workflow for multiway first split
 (define (multiway-model dataset field objective kind parameters)
   (let ([cs missings id name] (categories dataset field)
-        ds (for (c cs) (category-dataset dataset field c))
+        name ((fetch dataset) "name")
```

Comment:
i don't think that's right: your `name` here is shadowing the `name` returned by `(categories ..)`